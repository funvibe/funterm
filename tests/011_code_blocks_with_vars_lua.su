lua (greeting, message) {
    function greeting(hello)
        print(hello)
    end
    message = "hello again"
}
lua.greeting("hello from lua")
lua.print(lua.message)