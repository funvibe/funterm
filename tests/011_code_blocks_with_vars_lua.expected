hello from lua
hello again